src/components/tables/PoolsTable/TokenPills/TokenPills.vue

```diff
@@ -13,10 +13,12 @@ import HiddenTokensPills from './HiddenTokensPills.vue';
 type Props = {
   tokens: PoolToken[];
   isStablePool: boolean;
-  selectedTokens: string[];
+  selectedTokens?: string[];
```

Comment:
these are fixes from https://github.com/balancer-labs/frontend-v2/pull/844 (needs to be merged)